subsys/bluetooth/controller/hci_driver.c

```diff
@@ -24,7 +25,10 @@ static K_SEM_DEFINE(sem_signal, 0, UINT_MAX);
 static struct k_thread recv_thread_data;
 static struct k_thread signal_thread_data;
 static K_THREAD_STACK_DEFINE(recv_thread_stack, CONFIG_BLECTLR_RX_STACK_SIZE);
-static K_THREAD_STACK_DEFINE(signal_thread_stack, CONFIG_BLECTLR_SIGNAL_STACK_SIZE);
+static K_THREAD_STACK_DEFINE(signal_thread_stack,
+			     CONFIG_BLECTLR_SIGNAL_STACK_SIZE);
+
+static u8_t ble_controller_mempool[0x6000];
```

Comment:
Make it configurable?
we should probably ask the kernel for this memory. Can the alloc/init function be used to get the size now?

Reply:
We should, however not for this release of NCS. I have created an internal issue to follow this up.